buildSrc/src/test/groovy/io/radar/ci/GitHubActionsTest.groovy

```diff
@@ -11,6 +11,9 @@ class GitHubActionsTest {
     void testGet() {
         if (Boolean.parseBoolean(System.getenv('GITHUB_ACTIONS'))) {
             assert GitHubActions.get()
+            String snapshot = System.getenv 'SNAPSHOT'
+            //ensures that this variable is defined in GitHub Action workflows
```

Comment:
also ensures that it's a valid stringified-boolean

Reply:
Right. I could use `Boolean.parseBoolean`, however this does not check all needed scenarios. This ensures that the variable is defined, even if it is false.